src/components/TimeCardPage/SubmitCard.js

```diff
@@ -6,15 +6,27 @@ import Button from 'react-bootstrap/Button';
 
 
 
+
 export default function SubmitCard(props) {
 
     const [submitted, setSubmitted] = useState(false); 
     const [submitDate, setSubmitDate] = useState(null); 
+    const [state, setState] = useState('Rejected'); 
+    const [rejected, setRejected] = useState(true);
```

Comment:
@kennedyry Also getting these from the same GET timesheet? If so, see below comment about passing as prop

Reply:
Yep! It should probably be the whole comment object (Ill add it to the below comment also)